powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -698,14 +698,30 @@ void testFailedScaOtpMaxFailedAttemptsIdentityRestart() throws Exception {
         powerAuthClient.removeActivation(activationId, "test");
     }
 
+    @Test
```

Comment:
No, it is already implemented at `PowerAuthOnboardingTest#testOtpMaxFailedAttemptsReached`